src/configs/nats.rs

```diff
@@ -0,0 +1,26 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+#[cfg_attr(
+    feature = "config-schema",
+    derive(schemars::JsonSchema),
+    schemars(deny_unknown_fields)
+)]
+#[serde(default)]
+pub struct NatsConfig<'a> {
+    pub format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
+    pub disabled: bool,
+}
+
+impl<'a> Default for NatsConfig<'a> {
+    fn default() -> Self {
+        NatsConfig {
+            format: "[$symbol$name]($style)",
+            symbol: "",
```

Comment:
I think this should have a symbol by default.

Reply:
Yeah I'm not sure what to put here lol.

Reply:
I used an envelope for "messages"